src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import { ProgressBar } from "../../../../../components/client/ProgressBar";
+import styles from "./FindExposures.module.scss";
+
+export const FindExposures = () => {
+  const [scanProgress, setScanProgress] = useState(0);
+  const [scanFinished, setScanFinished] = useState(false);
+  const [checkingScanProgress, setCheckingScanProgress] = useState(false);
+  const router = useRouter();
+
+  const progressSteps = 6;
+  const maxProgress = 100;
+  const totalBreachesCount = 672;
+  const scannedBreachesCount = Math.ceil((672 * scanProgress) / 100);
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      const nextProgress = scanProgress + progressSteps;
+      setScanProgress(Math.min(nextProgress, maxProgress));
+
+      // Check the scan progress and if it’s finished.
+      if (!checkingScanProgress && !scanFinished) {
+        setCheckingScanProgress(true);
+        void fetch("/api/v1/user/welcome-scan/progress")
+          .then((response) => response.json())
+          .then((result) => {
+            if (result.status && result.status === "finished") {
+              setScanFinished(true);
+            }
+            setCheckingScanProgress(false);
+          })
+          .catch((_) => setCheckingScanProgress(false));
+      }
```

Comment:
@rhelmer Should we be only calling this route locally for dev?

Reply:
I think we still need to show the scan progress in either case, it's this part specifically where `getScanDetails` is called and then the results stored that we don't need to do in non-dev: https://github.com/mozilla/blurts-server/pull/3219/files#diff-2bb7d4c5a17c8bc653a51e2717446d7aeb2dc9be92711b8dfa642189b602ae13R48